content/docs/studio/user-guide/model-registry/assign-stage.md

```diff
@@ -1,18 +1,29 @@
-# Assign stages to models
+# Assign stage to model version
 
-To manage model lifecycle, you can assign stages (such as `development`,
-`staging`, `production`, etc.) to specific model versions.
+To manage model lifecycle, you can assign stages (such as `dev`, `staging`,
```

Comment:
The full names (eg, `development`) were used intentionally, so that it's clear to everyone. Do you think `dev` / `prod` are more common/understandable?

Reply:
Yes, I think they're popular enough and thus understandable.